app/classifier/frame-annotator.jsx

```diff
@@ -114,8 +102,6 @@ export default class FrameAnnotator extends React.Component {
           {!!AfterSubject && (
             <AfterSubject {...hookProps} />)}
 
-          <TooltipLayer screenCTM={screenCTM} />
```

Comment:
Yes, I've moved it to the SVG renderer - it's only designed to work with image subjects at the moment.